### PR TITLE
Solve issue: error output is truncated when contains "<?"

### PR DESCRIPTION
**Pull request by CharlesZ-Chen**
Solve issue https://github.com/eisop/webserver/issues/27.

### CODE REVIEW

File: static/js/cf-frontend.js [hunk 1]
```diff
@@ -451,9 +451,16 @@ function setErrorTable(error_report) {
     var error = error_report[errorIndex];
     //add error to the last pos of err table
     var row = error_table.insertRow(-1);
+
+    //Replace "\n" by html <br/> tag.
+    excetion_msg = error.exception_msg.replace(/\n/g,"<br/>&nbsp");
+    //Replace "<?" by "< ?" to avoid browser process it as html comment.
+    //Solve issue: https://github.com/eisop/webserver/issues/27
+    excetion_msg = excetion_msg.replace(/<\?/g, "< ?");
```
Comment (wmdietl) [inline static/js/cf-frontend.js]: Wouldn't it be safer to replace HTML characters in general? That would include `< > &` and probably a few others.

File: static/js/cf-frontend.js [hunk 2]
```diff
@@ -451,9 +451,16 @@ function setErrorTable(error_report) {
     var error = error_report[errorIndex];
     //add error to the last pos of err table
     var row = error_table.insertRow(-1);
+
+    //Replace "\n" by html <br/> tag.
+    excetion_msg = error.exception_msg.replace(/\n/g,"<br/>&nbsp");
```
Comment (wmdietl) [inline static/js/cf-frontend.js]: excetion -> exception

File: static/js/cf-frontend.js [hunk 3]
```diff
@@ -451,9 +451,16 @@ function setErrorTable(error_report) {
...
+    //Replace "<?" by "< ?" to avoid browser process it as html comment.
+    //Solve issue: https://github.com/eisop/webserver/issues/27
```
Comment (wmdietl) [inline static/js/cf-frontend.js]: Don't include the issue link or description of a previous bug. Describe what your code does and why.

**Comment by CharlesZ-Chen** [comment]
It turns out the original code base already has a function of processing html special chars...:

https://github.com/eisop/webserver/blob/master/static/js/cf-frontend.js#L568

Thus I just use this util function to escape general html special chars in our error outputs.

**Pull request merged**